assets/src/scss/new-identity/style.scss

```diff
@@ -52,6 +52,10 @@
   --block-spreadsheet-blue-header--background: #167f82;
   --block-spreadsheet-blue-odd-row--background: var(--beige-200);
   --block-spreadsheet-blue-even-row--background: var(--beige-100);
+  --link--color: var(--grey-900);
+  --link--visited--color: var(--grey-900);
+  --link--hover--color: var(--grey-900);
+  --link--hover--text-decoration: none;
```

Comment:
This also affects links that **should** have an underline on hover, such as the contextual links... That means we'll need a new CSS variable for those links